class.jetpack-debugger.php

```diff
@@ -4,7 +4,7 @@ class Jetpack_Debugger {
 
 	private static function is_jetpack_support_open() {
 		try {
-			$response = wp_remote_request( "http://jetpack.com/is-support-open" );
+			$response = wp_remote_request( "https://jetpack.com/is-support-open" );
```

Comment:
This brings up an interesting thing we have looked at yet with the broader dotcom move to all mapped domains to https. There are some servers that can pull http, but not https. When https was optional, we intentionally left this as http to allow the most servers possible able to ping us to determine if support was open.

Since last week's switch to all https, we lost this. Something we need to circle back with systems to determine if there's a way to exclude specific URLs from the autoredirect.